integration-test/it-runner/src/test/java/com/example/layrry/integrationtest/AbstractRemoteIntegrationTestCase.java

```diff
@@ -0,0 +1,59 @@
+/**
+ *  Copyright 2020 The ModiTect authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.example.layrry.integrationtest;
+
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.server.handler.DefaultHandler;
+import org.eclipse.jetty.server.handler.HandlerList;
+import org.eclipse.jetty.server.handler.ResourceHandler;
+import org.junit.AfterClass;
+
+import java.nio.file.Path;
+
+public abstract class AbstractRemoteIntegrationTestCase extends AbstractIntegrationTestCase {
```

Comment:
How about making this an `@Rule` (or whatever is the counterpart in JUnit 5) instead of a base class? It's fine for me as is, too, just wanting to suggest a more canonical approach, so to make sure we've discussed it.

Reply:
I thunk all the code found in this abstract class can be moved to a JUnit5 extension however its super class should also be turned into an extension, which contains assertion code.